pkg/volume/rbd/rbd.go

```diff
@@ -486,16 +521,95 @@ func (c *rbdUnmounter) TearDownAt(dir string) error {
 	return diskTearDown(c.manager, *c, dir, c.mounter)
 }
 
-func getVolumeSource(
-	spec *volume.Spec) (*v1.RBDVolumeSource, bool, error) {
+func getVolumeSourceMonitors(spec *volume.Spec) ([]string, error) {
+	if spec.Volume != nil && spec.Volume.RBD != nil {
+		mon := spec.Volume.RBD.CephMonitors
```

Comment:
for all the accessors, remove all the intermediate assignments and just `return spec.Volume.RBD.CephMonitors, nil` directly, etc

Reply:
done